src/index.scss

```diff
@@ -75,6 +75,21 @@ code {
   }
 }
 
+#viewUserProfilePage {
```

Comment:
Here we can extract the common styling of different forms. Please refer https://sass-lang.com/guide for more details.
```
%form {
  .MuiInput-formControl {
    .MuiInput-input.Mui-disabled {
      color: black;
      -moz-user-select: text;
    }

    svg {
      visibility: hidden;
    }
  }
}

#viewDictionaryPage {
  #dictionary-form {
    @extend %form;
  }
}

#viewSourcePage {
  #source-form {
    @extend %form;
  }
}

#viewUserProfilePage {
  #user-form {
     @extend %form;
   }
}
```